lib/chewy/rspec/helpers.rb

```diff
@@ -0,0 +1,55 @@
+module Chewy
+  module Rspec
+    module Helpers
+      extend ActiveSupport::Concern
+      # Rspec helper to mock elasticsearch response
+      # To use it - add `require 'chewy/rspec'` to the `spec_helper.rb`
+      #
+      #   mock_elasticsearch_response(CitiesIndex, raw_response)
+      #   expect(CitiesIndex.query({}).hits).to eq(hits)
+      #
+      def mock_elasticsearch_response(index, raw_response)
+        mocked_request = Chewy::Search::Request.new(index)
+        allow(Chewy::Search::Request).to receive(:new).and_return(mocked_request)
+        allow(mocked_request).to receive(:perform).and_return(raw_response)
+      end
+
+      # Rspec helper to mock Elasticsearch response source
+      # To use it - add `require 'chewy/rspec'` to the `spec_helper.rb`
+      #
+      #  mock_elasticsearch_response_sources(CitiesIndex, sources)
+      #  expect(CitiesIndex.query({}).hits).to eq(hits)
+      #
+      def mock_elasticsearch_response_sources(index, hits)
+        raw_response = {
+          'took' => 4,
+          'timed_out' => false,
+          '_shards' => {
+            'total' => 1,
+            'successful' => 1,
+            'skipped' => 0,
+            'failed' => 0
+          },
+          'hits' => {
+            'total' => {
+              'value' => hits.count,
+              'relation' => 'gte'
```

Comment:
Why `gte`?

Reply:
To be honest, we got this executing a simple query and using it as default scaffold without bothering much assuming that it would be a sane default.
Checking at ES documentation [here](https://www.elastic.co/guide/en/elasticsearch/reference/7.5/search-request-body.html#request-body-search-track-total-hits) it seems that this parameter is sent when total is approximate lower bound (the default behavior).
We could try to interpret `track_total_hits` value changing it to `eq` when appropriate, but I don't believe it worth the effort since we also provided a way to mock full response when more details are concerned.... wdyt @rabotyaga?

Reply:
According to documentation, the default for `track_total_hits` is `10_000`, meaning that the relation normally is `eq` for all results under 10 000. I believe it would be better to use `eq` here (unconditionally), and also change the `max_score` to `1.0`.